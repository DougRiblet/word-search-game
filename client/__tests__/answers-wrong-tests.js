import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import AnswersWrong from '../components/Answers-Wrong/answers-wrong-present';

describe('AnswersWrong', () => {
  test('Snapshot', () => {
    const dummystring = 'ABCD EFGH MNOP';
    const AnswersWrong_tree = renderer.create(
      <AnswersWrong wrongfour={dummystring} />
    ).toJSON();
    expect(AnswersWrong_tree).toMatchSnapshot();
  });

  test('renders without crashing', () => {
    const dummystring = 'ABCD EFGH MNOP';
    const div = document.createElement('div');
    ReactDOM.render(<AnswersWrong wrongfour={dummystring} />, div);
  });
});
