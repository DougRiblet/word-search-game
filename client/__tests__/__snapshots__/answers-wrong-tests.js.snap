// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnswersWrong Snapshot 1`] = `
<div
  id="answers-wrong"
>
  ABCD EFGH MNOP
</div>
`;
